CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## 2.3.0 (2019-02-06)
 ### Added
-* Add support for Azure IoT distributed tracing
+* Add support for [Azure IoT distributed tracing (preview)](https://aka.ms/iottracing)

```

Comment:
Azure IoT Hub distributed tracing (preview)

Reply:
I call `Azure IoT distributed tracing` due to the naming in https://review.docs.microsoft.com/en-us/azure/iot-hub/iot-hub-distributed-tracing?branch=pr-en-us-61655#understand-azure-iot-distributed-tracing-work-in-progress, and in the future distributed tracing may exist in other Azure IoT related service. Shall we follow the naming in docs?